src/Domain.LinnApps/ImportBooks/LedgerPeriod.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Linn.Stores.Domain.LinnApps.ImportBooks
+{
+    public class LedgerPeriod
```

Comment:
i'd move this out of the importbooks folder. definitely a general concern.

Reply:
Cool. On ledgerPeriod - I can't get it to run, it's complaining that it can't map the table into the repository. When I do desc ledger_periods, period_number is mapped as just a NUMBER, with no limit so guess that will be the max oracle size? I wondered after int was failing, so tried long and decimal but still not working. Do you know of anything we normally have used or is it likely I've just missed something else?

Reply:
Can't see anything obviously wrong with that. If you're still stuck i'll pull it down and have a look

Reply:
If you get a chance that'd be much appreciated, it seems like such a simple table and I can't see anything silly that's wrong with it so far. Wondered if it was that the sys/dev db was different but it's not that either

Reply:
i remember lewis mapped it as a view in the tax stuff. you could try that for now as we'll never update that table

Reply:
Was all a wild goose chase, sorry! I kept missing that I was trying to register LedgerPeriod to the IRepository, not LedgerPeriodRepository to it. Big time Doh. Seems happy as a table/dbset now though even with just an int